setup.py

```diff
@@ -26,17 +26,18 @@
         "License :: OSI Approved :: BSD License",
         "Operating System :: OS Independent",
     ],
-    python_requires=">=3.6",
+    python_requires=">=3.7",
     install_requires=[
-        "webstories>=0.0.1,<0.1",
-        "requests>=2.24.0,<3",
+        "wagtail>=4.1",
+        "webstories>=0.0.1",
+        "requests>=2.24.0",
```

Comment:
I'm not keen on dropping upper bounds from dependencies, because if one of these packages releases a backwards-incompatible major version in future, it will break existing projects that don't explicitly pin all their dependencies - and to me, breaking a configuration that previously worked without changes is a worse sin than leaving people stuck on old versions...